generate-master-bosh-certs.sh

```diff
@@ -25,3 +20,21 @@ rm ${TMPKEY}
 
 # upload it to AWS.  Use this as the default_key_name is all bosh manifests
 aws ec2 import-key-pair --key-name "masterbosh-$(date +'%Y%m%d')" --public-key-material "$(cat ${TARGET}/master-bosh.pub)"
+
+# generate a key and cert for master-bosh nginx
```

Comment:
We should eventually separate out CA generation from masterbosh deployment certs as part of secrets automation